packages/grid/_modules_/grid/utils/domUtils.ts

```diff
@@ -86,3 +76,21 @@ export function getGridCellElementFromIndexes(
     `:scope .${GRID_CELL_CSS_CLASS}[aria-colIndex='${colIndex}'][data-rowIndex='${rowIndex}']`,
   ) as HTMLDivElement;
 }
+
+export function getGridColumnHeaderElement(root: Element, field: string) {
+  return root.querySelector(
+    `:scope [role='columnheader'][data-field='${field}']`,
```

Comment:
Would it work without `:scope`? I don't get why we need this unless we have [a selector like `>`](https://developer.mozilla.org/en-US/docs/Web/CSS/:scope#direct_children). This seems already implicit:
```suggestion
    `[role='columnheader'][data-field='${field}']`,
```